README.md

```diff
@@ -1,7 +1,24 @@
-# Learning-Resource-Path-Front-End
+#  Learning-Resource-Path-Front-End
 
-## Beginner Level Front End Development
+##  Beginner Level Front End Development
 
-## Mid Level Front End Development
+### HTML
 
-## Advanced Level Front End Development
\ No newline at end of file
+ - [freeCodeCamp](https://learn.freecodecamp.org/)
+ - [Codeacademy](https://www.codecademy.com/)
+
+### CSS
```

Comment:
I think for this we don't need to repeat links. We can just add it under one section. or if there is a specific link for CSS in the resource add that instead. e.g  https://www.codecademy.com/learn/learn-css